modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -164,6 +181,77 @@ static Mat extractConstant(const Ptr<ImportGraphWrapper>& net, int node_id, int
     }
 }
 
+/*  Slice operator has two optional inputs "axes" and "steps". Some models may be set to have
+    Slice with optional inputs of default values, some of them don't. This Subgraph removes
```

Comment:
> some of them don't

Means two things - they still have such inputs but with different values or they don't have inputs at all.

Can we do matching in the opposite way - align all the Slice layers (with less than 5 inputs) to some canonical version with 5 inputs? Filling missed values with default ones.